packages/maskbook/src/utils/flags.ts

```diff
@@ -32,7 +32,8 @@ export const Flags = {
     //#region Experimental features
     image_payload_marked_as_beta: appOnly,
     /** Prohibit the use of test networks in production */
-    wallet_network_strict_mode_enabled: process.env.NODE_ENV === 'production' && !betaOrInsiderOnly,
+    wallet_network_strict_mode_enabled:
+        (process.env.NODE_ENV === 'production' && !betaOrInsiderOnly) || Math.random() < 1000,
```

Comment:
why need random?

Reply:
+1, this expression looks like `|| false` for me cause it always true.

Reply:
Oops, it's introduced for debugging. But it's OK to release it.